lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -352,7 +352,10 @@ def response_from_result(result)
           message_from_result(result),
           { braintree_transaction: transaction_hash(result) },
           { authorization: (result.transaction.id if result.transaction) }
-        )
+            authorization: (result.transaction.id if result.transaction),
+            test: test?
+          }
+         )
```

Comment:
to fix the failing test:
```ruby
      def response_from_result(result)
        response = { braintree_transaction: transaction_hash(result) }
        Response.new(
          result.success?, 
          message_from_result(result),
          response,
          authorization: (result.transaction.id if result.transaction),
          test: test?
        )
      end
```

Reply:
moved to https://github.com/activemerchant/active_merchant/pull/2980 to be able to edit